stability: panic in lightstep on register

sha: 04c996e
lightstep/lightstep-tracer-go sha: f3c66066ce6023ad1bd721cfbd9fbd6292eb08cc

Panic on node `ec2-54-152-207-175.compute-1.amazonaws.com`:
```
panic: runtime error: slice bounds out of range

goroutine 11 [running]:
panic(0x159dba0, 0xc420010130)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
bytes.(*Buffer).Write(0xc461eaa2a0, 0xc420428162, 0x8, 0x40, 0x0, 0x0, 0x0)
        /usr/local/go/src/bytes/buffer.go:135 +0xc4
github.com/lightstep/lightstep-tracer-go/thrift_0_9_2/lib/go/thrift.(*THttpClient).Write(0xc420144380, 0xc420428162, 0x8, 0x40, 0xc472700001, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/thrift_0_9_2/lib/go/thrift/http_client.go:216 +0x4d
github.com/lightstep/lightstep-tracer-go/thrift_0_9_2/lib/go/thrift.(*TBinaryProtocol).WriteI64(0xc420428120, 0x53be6c6f1b305, 0x10, 0x1000a)
        /go/src/github.com/lightstep/lightstep-tracer-go/thrift_0_9_2/lib/go/thrift/binary_protocol.go:207 +0xa9
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*LogRecord).writeField1(0xc476d56e00, 0x20858a0, 0xc420428120, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/ttypes.go:935 +0xe8
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*LogRecord).Write(0xc476d56e00, 0x20858a0, 0xc420428120, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/ttypes.go:885 +0x14f
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*SpanRecord).writeField10(0xc448334200, 0x20858a0, 0xc420428120, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/ttypes.go:1764 +0x12c
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*SpanRecord).Write(0xc448334200, 0x20858a0, 0xc420428120, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/ttypes.go:1604 +0x33f
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*ReportRequest).writeField3(0xc422575170, 0x20858a0, 0xc420428120, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/ttypes.go:2959 +0x12c
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*ReportRequest).Write(0xc422575170, 0x20858a0, 0xc420428120, 0x2000c, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/ttypes.go:2902 +0x18d
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*ReportArgs).writeField2(0xc435daf110, 0x20858a0, 0xc420428120, 0x0, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/reportingservice.go:337 +0xaf
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*ReportArgs).Write(0xc435daf110, 0x20858a0, 0xc420428120, 0x5806f00000001, 0x0)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/reportingservice.go:308 +0x18d
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*ReportingServiceClient).sendReport(0xc4200166e0, 0xc4200ce028, 0xc422575170, 0x1, 0xc422575170)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/reportingservice.go:74 +0xe5
github.com/lightstep/lightstep-tracer-go/lightstep_thrift.(*ReportingServiceClient).Report(0xc4200166e0, 0xc4200ce028, 0xc422575170, 0x10, 0x5, 0x5)
        /go/src/github.com/lightstep/lightstep-tracer-go/lightstep_thrift/reportingservice.go:54 +0x43
github.com/lightstep/lightstep-tracer-go.(*Recorder).Flush(0xc42037c000)
        /go/src/github.com/lightstep/lightstep-tracer-go/recorder.go:349 +0x19a3
github.com/lightstep/lightstep-tracer-go.(*Recorder).reportLoop(0xc42037c000)
        /go/src/github.com/lightstep/lightstep-tracer-go/recorder.go:467 +0x199
created by github.com/lightstep/lightstep-tracer-go.NewRecorder
        /go/src/github.com/lightstep/lightstep-tracer-go/recorder.go:224 +0x7f1
```

--- a/GLOCKFILE
+++ b/GLOCKFILE
@@ -78,7 +78,7 @@ github.com/kr/pretty cfb55aafdaf3ec08f0db22699ab822c50091b1c4
 github.com/kr/text 7cafcd837844e784b526369c9bce262804aebc60
 github.com/leekchan/timeutil 28917288c48df3d2c1cfe468c273e0b2adda0aa5
 github.com/lib/pq ae8357db35d721c58dcdc911318b55bef6b1b001
-github.com/lightstep/lightstep-tracer-go 7e238cc7deca88342f9fb7875ffc97cc450e6607
+github.com/lightstep/lightstep-tracer-go 91ae78ce073799b6a40e303d7dec6cc3a4257915
 github.com/mattn/go-isatty 66b8e73f3f5cda9f96b69efd03dd3d7fc4a5cdb8
 github.com/mattn/go-runewidth 737072b4e32b7a5018b4a7125da8d12de90e8045
 github.com/mattn/goveralls edf7508a2bcb40ec1160e94518c983c2fc169f02

--- a/pkg/util/tracing/tracer.go
+++ b/pkg/util/tracing/tracer.go
@@ -218,6 +218,7 @@ var newTracer = func() opentracing.Tracer {
 		lsTr := lightstep.NewTracer(lightstep.Options{
 			AccessToken:    lightstepToken,
 			MaxLogsPerSpan: maxLogsPerSpan,
+			UseGRPC:        true,
 		})
 		if lightstepOnly {
 			return lsTr

